tests/python/driver/tvmc/conftest.py

```diff
@@ -242,7 +242,7 @@ def tflite_mobilenet_v1_0_25_128(tmpdir_factory):
 
 @pytest.fixture(scope="session")
 def tflite_cnn_s_quantized(tmpdir_factory):
-    base_url = "https://github.com/ARM-software/ML-zoo/raw/master/models/keyword_spotting/cnn_small/tflite_int8/"
+    base_url = "https://github.com/ARM-software/ML-zoo/raw/master/models/keyword_spotting/cnn_small/model_package_tf/model_archive/TFLite/tflite_int8/cnn_s_quantized.tflite"
     file_to_download = "cnn_s_quantized.tflite"
     model_file = download_testdata(
         "{}/{}".format(base_url, file_to_download), file_to_download, module=["tvmc"]
```

Comment:
Not sure that it will work. After `"{}/{}".format(base_url, file_to_download)` we will get `https://.../tflite_int8/cnn_s_quantized.tflite/cnn_s_quantized.tflite`. So the url will be incorrect.

Reply:
Oops yes you're correct, thanks!